apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/hooks/use-keydown-handler.ts

```diff
@@ -4,30 +4,29 @@
  */
 import { useEffect, useCallback } from '@wordpress/element';
```

Comment:
Why are we importing these from `@wordpress/element`?

Reply:
Like we said in PM - we love our own abstractions! For posterity though, I think the official docs sum things up pretty well:

> You may find yourself asking, “Why an abstraction layer?”. For a few reasons:
> 
> - In many applications, especially those extended by a rich plugin ecosystem as is the case with WordPress, it’s wise to create interfaces to underlying third-party code. The thinking is that if ever a need arises to change or even replace the underlying implementation, it can be done without catastrophic rippling effects to dependent code, so long as the interface stays the same.
> - It provides a mechanism to shield implementers by omitting features with uncertain futures (createClass, PropTypes).
> - It helps avoid incompatibilities between versions by ensuring that every plugin operates on a single centralized version of the code.
> 

https://developer.wordpress.org/block-editor/reference-guides/packages/packages-element/

Of course, something holds until something better comes along! We evolve :-)

Reply:
Thanks so much for elaborating! This was a nice TIL-moment for me :)